### DISCUSSION TITLE
Unable to disable client access to advanced properties on the submit button

### DISCUSSION THREAD

**Discussion by thisisjamessmith** [solspace/craft-freeform#1513] [Feature Requests]
### What happened?

Thank you for the new "Limited Users" functionality. However it doesn't seem to go far enough. Take this monstrosity for example: 
<img width="233" alt="image" src="https://github.com/user-attachments/assets/bac71446-52ec-4bef-bedc-84ae9fdd866a">

Ignoring for a moment the inaccessible contrast ratios and using unintelligible icons for moving the button's position, in my opinion, no client anywhere should be expected to deal with these kind of decisions just for a submit button. They should, at most, be able to change the label text on the button.

Given that I've disabled the ability to do paginated forms in the "Limited Users" area, the 'Back' and 'Save' buttons shouldn't even be available.

### How can we reproduce this?

1. Create a Limited Users user group and disable access to Add Pages To Forms and disable Advanced Field Properties.
2. See that they can still attempt to change the position and attributes of the submit button (even though I'm using fully custom rendering, and have no intention of allowing them to change the position of the submit button)

### Environment

- Freeform Pro 5.x
- Craft 4.11.5

**Comment by kjmartens** [comment]
Hi @thisisjamessmith,

Thank you for your feedback on this.

This part of the form builder interface was not specifically intended to be hidden by any existing "Limited Users" settings. However, I can see how it would be good for us to add the ability to include this.

I have queued it up for development... likely sometime next week. 🙂

**Comment by kjmartens** [comment]
We have now added a check on some of the more advanced settings on Submit buttons in Freeform 5.6+ 🙂
